test/sections/error-handling/UC10/remove_problem_reducer_unit_test.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:the_process/actions/problems/remove_problem.dart';
+import 'package:the_process/models/app_state/app_state.dart';
+import 'package:the_process/models/navigation/page_data/problem_page_data.dart';
+import 'package:the_process/models/problems/problem.dart';
+import 'package:the_process/reducers/problems/remove_problem.dart';
+
+void main() {
+  group('RemoveProblemReducer', () {
+    final problem = Problem(errorString: 'Problem error message');
+    final problemPageData = ProblemPageData(problem: problem);
+    test(
+        'Removes Problem from appState.problems and ProblemPageData from appState.pagesData',
```

Comment:
```suggestion
        'removes Problem from appState.problems and ProblemPageData from appState.pagesData',
```

Uber nit - the string in group('... is the start of the sentence and the string in test('... is the continuation of the sentence so should start lower case.